pkg/apis/alerting/v1alpha2/register.go

```diff
@@ -0,0 +1,1380 @@
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/namespaces/policy").To(alerting.ModifyPolicyByAlertNamespace).
+		Doc("Modify Policy By Alert Namespace Level").
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(alerting.PolicyByAlert{}).
+		Writes(alerting.ModifyPolicyByAlertResponse{}).
+		Returns(http.StatusOK, RespOK, alerting.ModifyPolicyByAlertResponse{})).
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/namespaces/{ns_name}/policy").To(alerting.ModifyPolicyByAlertNamespace).
+		Doc("Modify Policy By Alert Namespace Level").
+		Param(ws.PathParameter("ns_name", "Specify namespace").DataType("string").Required(true).DefaultValue("")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(alerting.PolicyByAlert{}).
+		Writes(alerting.ModifyPolicyByAlertResponse{}).
+		Returns(http.StatusOK, RespOK, alerting.ModifyPolicyByAlertResponse{})).
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/namespaces/{ns_name}/workloads/policy").To(alerting.ModifyPolicyByAlertWorkload).
+		Doc("Modify Policy By Alert Workload Level").
+		Param(ws.PathParameter("ns_name", "Specify namespace").DataType("string").Required(true).DefaultValue("")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(alerting.PolicyByAlert{}).
+		Writes(alerting.ModifyPolicyByAlertResponse{}).
+		Returns(http.StatusOK, RespOK, alerting.ModifyPolicyByAlertResponse{})).
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/namespaces/{ns_name}/pods/policy").To(alerting.ModifyPolicyByAlertPod).
+		Doc("Modify Policy By Alert Pod Level").
+		Param(ws.PathParameter("ns_name", "Specify namespace").DataType("string").Required(true).DefaultValue("")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(alerting.PolicyByAlert{}).
+		Writes(alerting.ModifyPolicyByAlertResponse{}).
+		Returns(http.StatusOK, RespOK, alerting.ModifyPolicyByAlertResponse{})).
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/nodes/{node_id}/pods/policy").To(alerting.ModifyPolicyByAlertPod).
+		Doc("Modify Policy By Alert Pod Level").
+		Param(ws.PathParameter("node_id", "Specify node id").DataType("string").Required(true).DefaultValue("")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(alerting.PolicyByAlert{}).
+		Writes(alerting.ModifyPolicyByAlertResponse{}).
+		Returns(http.StatusOK, RespOK, alerting.ModifyPolicyByAlertResponse{})).
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.PATCH("/namespaces/{ns_name}/pods/{pod_name}/containers/policy").To(alerting.ModifyPolicyByAlertContainer).
+		Doc("Modify Policy By Alert Container Level").
+		Param(ws.PathParameter("ns_name", "Specify namespace").DataType("string").Required(true).DefaultValue("")).
```

Comment:
I think `namespace` would be a better name

Reply:
Fixed.